examples/webgl_camera_array.html

```diff
@@ -36,13 +36,14 @@
 
 					for ( var x = 0; x < AMOUNT; x ++ ) {
 
+						var xx = x / AMOUNT;
+						var yy = 1 - ( ( y + 1 ) / AMOUNT ); // TODO Make this saner in WebGLRenderer?
+
 						var subcamera = new THREE.PerspectiveCamera( 40, ASPECT_RATIO, 0.1, 10 );
-						subcamera.bounds = new THREE.Vector4( x / AMOUNT, y / AMOUNT, SIZE, SIZE );
-						subcamera.position.x = ( x / AMOUNT ) - 0.5;
-						subcamera.position.y = 0.5 - ( y / AMOUNT );
-						subcamera.position.z = 1.5;
-						subcamera.position.multiplyScalar( 2 );
-						subcamera.lookAt( new THREE.Vector3() );
+						subcamera.bounds = new THREE.Vector4( xx, yy, SIZE, SIZE );
+						subcamera.position.x = ( (AMOUNT - 1) / 2 - x) * 0.2;
+						subcamera.position.y = ( (AMOUNT - 1) / 2 - y) * 0.2;
+						subcamera.position.z = 3;
 						subcamera.updateMatrixWorld();
```

Comment:
I see `updateMatrixWorld()` is required here... So the subcamera's world matrix is not updated by the renderer? Is that the desired behavior?